export/index.template.html

```diff
@@ -709,6 +836,11 @@ <h2>Message log</h2>
       document.addEventListener(
         "DOMContentLoaded",
         async () => {
+          // If styles are saved in local storage, sanitize and apply them.
```

Comment:
You may have already done this, and nothing indicates the contrary, but would be nice to do a sanity check to make sure these loaded settings can be overridden if the user calls `applySettings` again

Reply:
Yes this works as exepected!